### ISSUE TITLE
[Send invoices] Confirmation modal doesn't close after confirmation

### ISSUE THREAD

**Issue by drummer83** [openfoodfoundation/openfoodnetwork#12803]
## Description

Sending invoices on the orders list page requires a confirmation of the user in a confirmation modal. However, this modal doesn't close after clicking confirm. Also clicking cancel or outside of the modal doesn't close it. The page has to be reloaded to get rid of it.

## Expected Behavior

The modal should close after clicking 'Confirm'.

## Actual Behaviour

The modal doesn't close after clicking 'Confirm'.

## Steps to Reproduce

1. Go to /orders page.
2. Select one or more orders.
3. Select 'Send invoices' in the actions dropdown menu.
4. Click 'Confirm'.
5. Note that the emails are being sent, but the modal doesn't close.

## Animated Gif/Screenshot

[Kazam_screencast_00012.webm](https://github.com/user-attachments/assets/e23c8535-4bd2-4687-85ba-b5b0818868ef)

## Workaround

Reload page.

## Severity
bug-s4: it's annoying, but you can use it

## Your Environment

* Version used:
* Browser name and version:
* Operating System and version (desktop or mobile):

**Comment by sigmundpetersen** [comment]
@chahmedejaz will this also be closed by #13004 ?

**Comment by chahmedejaz** [comment]
Yes, @sigmundpetersen - It will get fixed by that PR. That's impressive how you are always being so observant. :smile: :hearts:
